nixos/modules/programs/steam.nix

```diff
@@ -24,15 +45,31 @@ in {
           ];
         }
       '';
-      apply = steam: steam.override (prev: {
-        extraLibraries = pkgs: let
-          prevLibs = if prev ? extraLibraries then prev.extraLibraries pkgs else [ ];
-          additionalLibs = with config.hardware.opengl;
-            if pkgs.stdenv.hostPlatform.is64bit
-            then [ package ] ++ extraPackages
-            else [ package32 ] ++ extraPackages32;
-        in prevLibs ++ additionalLibs;
-      });
+      apply = steam:
+        steam.override (
+          prev:
+            {
+              extraLibraries = pkgs: let
+                prevLibs =
+                  if prev ? extraLibraries
+                  then prev.extraLibraries pkgs
+                  else [];
+                additionalLibs = with config.hardware.opengl;
+                  if pkgs.stdenv.hostPlatform.is64bit
+                  then [package] ++ extraPackages
+                  else [package32] ++ extraPackages32;
+              in
+                prevLibs ++ additionalLibs;
+            }
+            // optionalAttrs
+            (cfg.gamescopeSession.enable && gamescopeCfg.capSysNice)
+            {
+              buildFHSEnv = pkgs.buildFHSEnv.override {
+                # use the setuid wrapped bubblewrap
+                bubblewrap = "${config.security.wrapperDir}/..";
+              };
+            }
+        );
```

Comment:
Can you please revert the formatting changes here? They make reviewing harder and I don't actually know if something here changed or not on a glance.